tests/Unit/Momento.Sdk.Tests/CacheClientConstructorTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Momento.Sdk;
+using Momento.Sdk.Auth;
+using Momento.Sdk.Config;
+using Xunit;
+
+public class CacheClientConstructorTest
+{
+    const String TEST_AUTH_TOKEN = "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJmb29Abm90LmEuZG9tYWluIiwiY3AiOiJjb250cm9sLXBsYW5lLWVuZHBvaW50Lm5vdC5hLmRvbWFpbiIsImMiOiJjYWNoZS1lbmRwb2ludC5ub3QuYS5kb21haW4ifQo.rtxfu4miBHQ1uptWJ2x3UiAwwJYcMeYIkkpXxUno_wIavg4h6YJStcbxk32NDBbmJkJS7mUw6MsvJNWaxfdPOw";
```

Comment:
![](https://media.giphy.com/media/29SqSyXlyO6WI/giphy.gif)

Reply:
this token isn't actually valid.  But I can make that more clear in the variable name / comments.

Reply:
What's not valid about it? Could we replace it with a string that just says "IntentionallyFakeAuthToken"?

Reply:
I can change the variable name to that.  I can't actually test the constructor without a string that can at least be deserialized without error though.

Reply:
> what's not valid

it has bogus URLs and the signature is fake.

Reply:
added some clarification in https://github.com/momentohq/client-sdk-dotnet/pull/408/commits/6beab8fbcf85510f9fdff6e23530c5bce3448515